src/components/elementView/rackTab.tsx

```diff
@@ -142,7 +142,7 @@ class RackTab extends React.Component<RackTabProps, RackTabState> {
       loading: true
     });
     axios
-      .get(API_ROOT + "api/racks/get-all", getHeaders(this.props.token))
+      .post(API_ROOT + "api/racks/get-many", { sort_by: [], filters: [] }, getHeaders(this.props.token))
```

Comment:
what endpoint are you calling? i thought we only had get-all or get, but did something get added but not updated in the endpoints table?